modules/variantbam/main.nf

```diff
@@ -25,7 +25,6 @@ process VARIANTBAM {
 
     output:
     tuple val(meta), path("*.bam")         , emit: bam
-    tuple val(meta), path("*.qcreport.txt"), optional: true, emit: qcreport
```

Comment:
I have decided that it might be best to not actually add this, the reason being is that I feel it would be rarely used for this but it requires an additional input to always be passed given the current module setup (optional `val` as an `input`).  Given that there is `samtools/stats` which even has a its own `multiqc module` I feel like it might be best to not include this optional input as it is catering to very rare usage at the expense of an additional input.